### PR TITLE
Cards vue actions

### PR DESCRIPTION
**Pull request by chosta**
* Some action cards
* More symbols and items on cards

Note: Instead of dealing with painful abstractions by creating CardAction classes, I decided to just modify the `description()` function to accept the full text `Action:...` or `Effect...`

### CODE REVIEW

File: src/cards/AICentral.ts [hunk 1]
```diff
@@ -6,31 +6,36 @@ import {Game} from '../Game';
...
+  }
+  public action(player: Player, game: Game) {
+    player.cardsInHand.push(game.dealer.dealCard(), game.dealer.dealCard());
+    return undefined;
+  }
+  public metadata: CardMetadata = {
+    description: 'Requires 3 Science tags to play. Decrease your Energy production 1 step',
+    cardNumber: '208',
+    requirements: CardRequirements.builder((b) => b.tag(Tags.SCIENCE, 3)),
+    renderData: CardRenderer.builder((b) => b.effectBox((ab) => ab.empty().startAction.cards(2).description('Action: Draw 2 cards')).br.productionBox((pb) => pb.minus().energy(1))),
```
Comment (kberg) [inline src/cards/AICentral.ts:39]: This is a bit hard to read, consider:

```
CardRenderer.builder((b) => {
  b.effectBox((ab) => ab.empty().startAction.cards(2).description('Action: Draw 2 cards'));
  b.br;
  b.productionBox((pb) => pb.minus().energy(1));
});
```

Comment (kberg) [reply src/cards/AICentral.ts:39]: (It doesn't all have to chain. :D)

Comment (chosta) [reply src/cards/AICentral.ts:39]: omg, you can do that...Where can I read about these patterns? It seems that there is hidden knowledge I'm not aware of..

File: src/cards/ares/BioengineeringEnclosure.ts [hunk 2]
```diff
@@ -9,60 +9,72 @@ import {Tags} from '../Tags';
...
+        },
+    ));
+    return undefined;
+  }
+  public metadata: CardMetadata = {
+    description: 'Requires 1 science tag to play. Add 2 animals to this card. OTHERS MAY NOT REMOVE ANIMALS FROM THIS CARD',
+    cardNumber: 'A01',
+    requirements: CardRequirements.builder((b) => b.tag(Tags.SCIENCE)),
+    renderData: CardRenderer.builder((b) => {
+      b.effectBox((eb) => eb.animals(1).asterix().startAction.animals(1).asterix().description('Action: Remove 1 animal from THIS card to add 1 animal to ANOTHER card')).br;
```
Comment (kberg) [inline src/cards/ares/BioengineeringEnclosure.ts:76]: Consider here and elsewhere

```
      b.effectBox((eb) => 
        eb.animals(1).asterix().startAction.animals(1).asterix()
        .description('Action: Remove 1 animal from THIS card to add 1 animal to ANOTHER card')).br;
```

Of course, a lot depends on the formatter. I would even argue for 

```
b.effectBox((eb) => 
  eb
    .animals(1)
    .asterix()
    .startAction
    .animals(1)
    .asterix()
    .description('Action: Remove 1 animal from THIS card to add 1 animal to ANOTHER card')).br;
```

File: src/cards/colonies/AtmoCollectors.ts [hunk 3]
```diff
@@ -10,54 +10,69 @@ import {IResourceCard} from '../ICard';
...
+  public metadata: CardMetadata = {
+    description: 'Add 2 floaters to ANY card',
+    cardNumber: 'C03',
+    renderData: CardRenderer.builder((b) => {
+      b.effectBox((eb) => {
+        eb.empty().startAction.floaters(1).description('Action: Add one floater here');
+      }).br;
+      b.or(CardRenderItemSize.SMALL).br;
+      b.effectBox((eb) => {
+        eb.floaters(1).startAction.titanium(2).digit.slash(CardRenderItemSize.SMALL).energy(3).digit.slash(CardRenderItemSize.SMALL).heat(4).digit.description('Action: Spend 1 floater here to gain 2 titanium, or 3 energy, or 4 heat');
```
Comment (kberg) [inline src/cards/colonies/AtmoCollectors.ts:73]: Same here, for instance.

**Review by kberg — APPROVED** [review]
So other than improving the readability of the renderData chains, this looks good. Good work!

Can you do something about the readability now? Then I'll pull it in.

File: src/cards/colonies/AtmoCollectors.ts (hunk 3 above)
Comment (chosta) [reply src/cards/colonies/AtmoCollectors.ts:73]: Yes, I also like the way things started to work. For now I'll stick to everything that new requires a new row on the card to be on a new row in the API. 
Thanks for the guidance.This is working pretty well, although I was a bit skeptical at the start :)

**Pull request merged**